html_app/ui/microscopy.soy

```diff
@@ -285,8 +285,9 @@
         </thead>
         {foreach $r in $rows}
         <tr  role='row'  class='scb_s_microscopy_samples_table_tr {if $r.display_sample} scb_s_microscopy_row_border{/if}'>
-            <td class='scb_s_microscopy_sample_checkbox scb_s_microscopy_samples_table_td {if $r.display_sample}{else}scb_s_experiment_setup_table_border{/if}' >{if $r.display_sample}
-					{if $r.cell_treatment.treatment_list.list[0].microscope[0] == 'na'}
+            <td class='scb_s_microscopy_sample_checkbox scb_s_microscopy_samples_table_td {if $r.display_sample}{else}scb_s_experiment_setup_table_border{/if}' >
+            {if $r.display_sample}
+					{if $r.cell_treatment.treatment_list.first.microscope[0] == 'na'}
```

Comment:
nit: spacing here seems weird, why so much indentation and/or lack of indentation for that first if?